openssl/src/x509/verify.rs

```diff
@@ -139,4 +150,16 @@ impl X509VerifyParamRef {
             .map(|_| ())
         }
     }
+
+    /// Set the verification time
+    #[corresponds(X509_VERIFY_PARAM_set_time)]
+    pub fn set_time(&mut self, time: time_t) {
```

Comment:
If nothing else, this should at least document the units of `time`.

Reply:
The format of time_t is not defined in the C standard, per https://en.cppreference.com/w/c/chrono/time_t, however it is usually seconds since epoch. I'm not sure we can reliably state that that will always be true in Rust, though

Reply:
Updated to add additional description to the comment